### DISCUSSION TITLE
Nano V1.2 klipper.bin file only 24108 bytes?

### DISCUSSION THREAD

**Discussion by roman-romaniuk** [Guilouz/Klipper-Flsun-Speeder-Pad#89] [Q&A]
Hi All,
I have an FLSun v400 and am attempting to reflash with the official klipper.  No issues upgrading the pad, but when I build the klipper.bin file for my Nano V2.1, I get a file size of ~ 24K, which doesn't seem right, especially given that the elf file is almost a meg, which seems to be more reasonable

pi@speeder-pad:~/klipper/out$ ls -l
total 1233
-rw-rw-r-- 1 pi pi  24108 Apr  2 10:25 Robin_nano35.bin
-rw-rw-r-- 1 pi pi   9273 Apr  2 10:10 autoconf.h
lrwxrwxrwx 1 pi pi     26 Apr  2 10:10 board -> /home/pi/klipper/src/stm32
drwxrwxr-x 2 pi pi   1024 Apr  2 10:10 board-generic
-rw-rw-r-- 1 pi pi     27 Apr  2 10:10 board-link
-rw-rw-r-- 1 pi pi  58400 Apr  2 10:11 compile_time_request.c
-rw-rw-r-- 1 pi pi    353 Apr  2 10:11 compile_time_request.d
-rw-rw-r-- 1 pi pi 165508 Apr  2 10:11 compile_time_request.o
-rw-rw-r-- 1 pi pi  13993 Apr  2 10:11 compile_time_request.txt
-rwxrwxr-x 1 pi pi  24108 Apr  2 10:11 klipper.bin
-rw-rw-r-- 1 pi pi   7562 Apr  2 10:11 klipper.dict
-rwxrwxr-x 1 pi pi 987252 Apr  2 10:11 klipper.elf
drwxrwxr-x 4 pi pi   1024 Apr  2 10:10 lib
drwxrwxr-x 4 pi pi   3072 Apr  2 10:11 src

any ideas on what I've done wrong? Or is the 24K file size correct?

Thanks,

Roman

**Comment by Guilouz** [comment]
Size is correct

**Reply by roman-romaniuk** [reply]
thanks for the amazingly fast response!!
